src/tray.js

```diff
@@ -16,14 +16,16 @@ const { CONFIG_KEY: PUBSUB_KEY } = require('./enable-pubsub')
 const { CONFIG_KEY: NAMESYS_PUBSUB_KEY } = require('./enable-namesys-pubsub')
 const { CONFIG_KEY: AUTO_GC_KEY } = require('./automatic-gc')
 const { CONFIG_KEY: AUTO_LAUNCH_WEBUI_KEY } = require('./webui')
+const { CONFIG_KEY: ASK_OPENING_IPFS_URIS } = require('./protocol-handlers')
```

Comment:
it looks like you're following convention, but config keys should all be defined in the same place, can you move us towards better practices by putting your config key into `src/common/configKeys.js` ?

Reply:
We can do that, would be great. However, outside of this PR, which is already a bit big.

Reply:
sounds fair!